scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts

```diff
@@ -340,7 +341,8 @@ export async function hasPurchaseableCollateral(comet: CometInterface, assets: A
   for (const asset of assets) {
     const collateralReserves = await comet.getCollateralReserves(asset.address);
     const price = await comet.getPrice(asset.priceFeed);
-    const value = collateralReserves.toBigInt() * price.toBigInt() / asset.scale;
+    const priceScale = exp(1, 8);
```

Comment:
Should we get this value on-chain just in case there's a Comet with a different price scale?

Reply:
great point

Reply:
although `PRICE_FEED_DECIMALS` is a constant right now:

```
uint8 internal constant PRICE_FEED_DECIMALS = 8;
```

So it's unlikely to change unexpectedly

Reply:
That's true, most likely won't change now that we went down the path of using wrapper price feeds instead.